youtube_music_playlist_downloader.py

```diff
@@ -71,7 +71,8 @@ def generate_metadata(file_path, link, album_name, track_num):
             info_dict = ytdl.extract_info(link, download=False)
             video_id = info_dict.get("id", None)
             video_title = info_dict.get("title", None)
-            uploader = info_dict.get("uploader", None)
+            uploader = info_dict.get("artist", None) or info_dict.get("uploader", None)
```

Comment:
Option for only "uploader"? (In regards to previous comment 'it can list the original artist instead of the cover artist')
Example cover that has this behavior (after adding to a playlist): https://www.youtube.com/watch?v=seu2gL0JP9k
'Music in this video' attributes to the original creator without listing the cover singer.

Reply:
> Option for only "uploader"? 

completely forgot about that :sweat_smile: 

> Wondering if I should store some playlist metadata in a file in the playlist folder so you don't need to re-enter options when updating.

Tbh i don't really mind, every added option should default to uploader/playlist name and i probably won't update playlists 99% of the time xD